src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -83,4 +87,37 @@ public DissolutionGetResponse getDissolutionApplication(
                 .get(companyNumber)
                 .orElseThrow(DissolutionNotFoundException::new);
     }
+
+    @Operation(summary = "Patch Dissolution Application", tags = "Dissolution")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Dissolution Application patched", content = @Content),
+            @ApiResponse(responseCode = "404", description = "Dissolution Application not found"),
+            @ApiResponse(responseCode = "409", description = "Dissolution Request already approved by this director", content = @Content)
```

Comment:
by doing `content = @Content`, the swagger docs will automatically assume that the 409 response will contain DissolutionPatchResponse. It won't. Remove this to be the same as 404